concreep-refilled/control.lua

```diff
@@ -1,14 +1,16 @@
-MINIMUM_ROBOTS = 30
-MINIMUM_ITEM_COUNT = 200
+SURFACES = settings.global["surface concreep is running on"].value

```

Comment:
Just to move the default constants to the top of the settings.lua file instead of here, so that they can be used in the settings file to set the defaults there (as well as instead of having to scroll through the data struct to change them).

As for lots of options, I don't think that's really a big deal. Plenty of "simpler" addons have long lists of options. :-/